WORKSPACE

```diff
@@ -36,5 +36,35 @@ upb_deps()
 
 # Load prometheus C++ dependencies.
 load("@com_github_jupp0r_prometheus_cpp//bazel:repositories.bzl", "prometheus_cpp_repositories")
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+http_archive(
+    name = "rules_foreign_cc",
+    sha256 = "69023642d5781c68911beda769f91fcbc8ca48711db935a75da7f6536b65047f",
+    strip_prefix = "rules_foreign_cc-0.6.0",
+    url = "https://github.com/bazelbuild/rules_foreign_cc/archive/0.6.0.tar.gz",
+)
+
+load("@rules_foreign_cc//foreign_cc:repositories.bzl", "rules_foreign_cc_dependencies")
```

Comment:
Nit: Can we use `rules_foreign_cc` to install boost library too - if that helps to build for Windows. Probably, we can create a github issue for Windows support, for someone to pick it up.

Reply:
Sure, I'll raise another PR for Windows support.